easybuild/easyconfigs/q/QBOX/QBOX-r140b_fix.patch

```diff
@@ -0,0 +1,139 @@
+# repair settings and code
+#Martin Golasowski, martin.golasowski@vsb.cz, 2017 IT4Innovations
+Only in qball_r140_working/examples/mgo_benchmark: mgo.N96.i
+Only in qball_r140_working/examples/mgo_benchmark: mgo.N96.out
+Only in qball_r140_working/examples/mgo_benchmark: mgo.N96.sys
+Only in qball_r140_working/: QBOX.patch
+diff -ru qball_r140/trunk/src/Context.C qball_r140_working/trunk/src/Context.C
+--- qball_r140/trunk/src/Context.C	2013-04-18 20:31:13.000000000 +0200
++++ qball_r140_working/trunk/src/Context.C	2017-03-10 12:19:09.026641211 +0100
+@@ -89,9 +89,9 @@
+ int Cblacs_pnum(int icontxt, int prow, int pcol)
+ { return 0;}
+ 
+-int Csys2blacs_handle(int comm) {
+-  return 0;
+-}
++//int Csys2blacs_handle(MPI_Comm* comm) {
++//  return 0;
++//}
+ void Cdgesd2d(int icontxt,int m,int n,double *A,int lda,int rdest,int cdest)
+ { return; }
+ 
+@@ -613,11 +613,12 @@
+   char order = 'C';
+   MPI_Comm_size(comm_,&nprocs);
+   MPI_Comm_rank(comm_,&mype_);
+-  int bhandle = Csys2blacs_handle(comm_);
++  //int bhandle = Csys2blacs_handle(comm_);
++  int bhandle = 0;
```

Comment:
MPI_Comm is a typedef struct ompi_communicator_t *MPI_Comm. That function Csys2blacs_handle expects int, is called only once in this place and returns a hardcoded zero. So, this change should not alter behavior, otherwise it will not build.